plugins/available/nvm.plugin.bash

```diff
@@ -5,7 +5,7 @@
 cite about-plugin
 about-plugin 'node version manager configuration'
 
-export NVM_DIR="$HOME/.nvm"
+[ -v $NVM_DIR ] && NVM_DIR="$HOME/.nvm"
```

Comment:
You dropped the `export` part of the statement - this might be important.

This could also be written like this:

```
export NVM_DIR=${NVM_DIR:-$HOME/.nvm}
```